internal/pkg/cli/storage_init.go

```diff
@@ -97,6 +138,11 @@ type initStorageVars struct {
 	lsiSorts     []string // lsi sort keys collected as "name:T" where T is one of [SNB]
 	noLSI        bool
 	noSort       bool
+
+	// RDS Aurora Serverless specific values collected via flags or prompts
+	engine         string
+	parameterGroup string
+	initialDBName  string
```

Comment:
my two cents: compared with using comments to tell which consts/member variables are RDS/DDB specific, I think it would be better to tell from the const/var names from users' perspective (otherwise I would constantly going back and forth to where they are defined to see what category they belong).

_Don't be afraid to make a name long. A long descriptive name is better than a short enigmatic name. A long descriptive name is better than a long descriptive comment. --Robert C. Martin_

Reply:
This sounds good to me! I think this should be done in a separate PR that focuses on refactoring. 

Just a note to myself: in the refactor PR we can also refactor [this](https://github.com/aws/copilot-cli/pull/2080#discussion_r597968279). Although the pointers in `newRDS` have been removed, `newS3` and `newDynamoDB` can probably benefit from the same refactoring too.